docs/pages/utilities.md

```diff
@@ -24,3 +24,9 @@ want to write these yourself but might need logging or a command-line interface.
 | --------------------------------------------------------- | -------------------------------------------------------------------- | :-: |
 | [loguru](https://loguru.readthedocs.io/)                  | Simple and user-friendly with many nice features enabled by default. | 🟢  |
 | [logging](https://docs.python.org/3/library/logging.html) | Python's builtin logging framework. Needs some configuration.        | 🟠  |
+
+## User interface
+
+| Name                                         | Short description       | 🚦  |
+| -------------------------------------------- | ----------------------- | :-: |
+| [tqdm](https://pypi.org/project/tqdm/2.2.3/) | Progress bar for loops. | 🟢  |
```

Comment:
```suggestion
| [tqdm](https://tqdm.github.io/) | Progress bar for loops. | 🟢  |
```
If linking to PyPI page then I think version non-specific `https://pypi.org/project/tqdm/` is probably better, but overall link to documentation site is more consistent with other link we have on this page.